README.md

```diff
@@ -65,6 +65,10 @@ on [Filecoin Slack](https://filecoinproject.slack.com/).
 
 6. Check logs with `docker logs -f saturn-node`
 7. Check there are no errors, registration will happen automatically and node will restart once it receives its TLS certificate
+
+In most intsances speedtest does a good job of picking "close" servers but for small networks it may be incorrect.
+If the speedtest value reported by speedtest seems low, you may want to configure SPEEDTEST_EXTRA to point to a different public speedtest server (e.g. `SPEEDTEST_EXTRA="-s 6468"` selects the Tekify Fiber & Wireless speedtest server in Fremont, CA).
```

Comment:
The example is not clear into how to get the server ID, is there a public list?

Reply:
Good point, so you can get the list from the speedtest command (e.g. `speedtest --list`) or by going to https://c.speedtest.net/speedtest-servers-static.php

There is also the gist ( https://gist.github.com/ofou/654efe67e173a6bff5c64ba26c09d058 ) but it might be out of date.